docs/how-to-work-on-tutorials-that-use-coderoad.md

```diff
@@ -0,0 +1,123 @@
+## How the tutorials work
+The freeCodeCamp tutorials that use CodeRoad each have their own repo under the freeCodeCamp GitHub organization. They all start with `learn-`. For example, `https://github.com/freeCodeCamp/learn-bash-by-building-a-boilerplate/`.
+
+Each tutorial repo has a `main` branch and a "version" branch, e.g. `v1.0.0`.
+
+The two main files on the `main` branch are `TUTORIAL.md` and `coderoad.yaml`. `TUTORIAL.md` contains all the instructions, hints, titles, and so on, for the tutorial. `coderoad.yaml` contains instructions for CodeRoad, such as what commands to run and when, what files to watch for changes, and what version branch to use for the steps.
+
+The "version" branch contains the commits that will be loaded on each step of a tutorial. The commit messages on this branch have to be specific. The first commit needs `INIT` for its message and contains all the files to load before the first lesson.
+
+Subsequent commit messages have to match the step number in `TUTORIAL.md` from the `main` branch. For example, the commit with the message `10.1` will be loaded when a user goes to step `10.1`.
+
+In order to make changes to commits on a version branch, you would need to rebase and edit the commits you want to change. This will rewrite the Git history, so we cannot accept PRs to these types of branches. Once a version branch is on the freeCodeCamp repo, it should never change.
+
+> [!WARNING]
+>
+> Never make or push changes to a version branch that is on one of the freeCodeCamp repos. Always create a new one
+
+## How to contribute
+
+### Prerequisites
+Install the [CodeRoad CLI tools](https://www.npmjs.com/package/@coderoad/cli) with `npm install -g @coderoad/cli`.
+
+There have been some issues with the latest version. If `coderoad --version` doesn't work after installing, downgrade to `0.7.0` with `npm install -g @coderoad/cli@0.7.0`.
+
+### Working on `main`
+This set of instructions is for PRs that only make minor changes on `main` to **existing lessons**. That mainly consists of typo, grammar, hint, and instructional changes or fixes in the `TUTORIAL.md` file.
```

Comment:
```suggestion
This set of instructions is for PRs that only make minor changes on `main` to **existing lessons**. That mainly consists of typos, grammar, hints, and instructional changes or fixes in the `TUTORIAL.md` file.
```

Reply:
I dunno - I think this reads better how it is